src/main/java/com/google/sps/servlets/MailServlet.java

```diff
@@ -79,10 +82,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 * This function creates an email and sends it to the user
 **/
 
-  private String sendMultipartMail(String from, String to, String link) {
+  private String sendMultipartMail(String from, String to, String title, String message, String image) {
     Properties props = new Properties();
     Session session = Session.getDefaultInstance(props, null);
-
+    final String POSTCARD_CONTAINER = "<div class='pcard-container' id='pcard-design' " +
+                "style='background-attachment: scroll; " +
+                "background-image: url(\"https://i.ibb.co/JjqsjjL/postcard.jpg\"); " +
+                "background-repeat: no-repeat; background-size: 700px 500px; color: black; " +
+                "display: block; height: 500px; margin-left: auto; margin-right: auto ;" +
+                "position: relative; text-align: center; width: 700px;'>";
+    
     try {
       Message msg = new MimeMessage(session);
```

Comment:
`msg` might be more easily understood shorthand, but I think the saving of characters by omitting vowels is not worth it

Reply:
Done